umh.docs.umh.app/content/en/docs/production-guide/administration/reduce-database-size.md

```diff
@@ -44,31 +44,92 @@ Connect to the `factoryinsight` database:
 \c factoryinsight
 ```
 
+Or connect to the `umh_v2` database:
+
+```bash
+\c umh_v2
+```
+
 ## Enable data compression
 
-To enable data compression, you need to execute the following SQL command from
-the database shell:
+You can find sample SQL commands to enable data compression here:
 
-```sql
-SELECT add_retention_policy('processvaluetable', INTERVAL '7 days');
-```
+### Sample command for the processvaluetable in the factoryinsight database:
+
+1. First, turn on compression:
+
+    ```sql
+    ALTER TABLE processvaluetable SET (timescaledb.compress, timescaledb.compress_segmentby = 'asset_id', timescaledb.compress_orderby = 'valuename');
+    ```
+    This command set `asset_id` as the key for the compressed segments 
+    and orders the table by `valuename`.
+
+2. Then, create a compression policy:
+
+    ```sql
+    SELECT add_compression_policy('processvaluetable', INTERVAL '7 days');
+    ```
+
+    This command will set a compression policy on the `processvaluetable` table,
+    which will compress data older than 7 days.
+
+### Sample command for the tag table in the umh_v2 database:
+
+1. First, turn on compression:
+
+    ```sql
+    ALTER TABLE tag SET (timescaledb.compress, timescaledb.compress_segmentby = 'asset_id', timescaledb.compress_orderby = 'name');
+    ```
+    This command set `asset_id` as the key for the compressed segments 
+    and orders the table by `name`.
+
+2. Then, create a compression policy:
+
+    ```sql
+    SELECT add_compression_policy('tag', INTERVAL '2 weeks');
+    ```
+
+    This command will set a compression policy on the `tag` table,
+    which will compress data older than 2 weeks.
+
+
+Refer to [the official documentation](https://docs.timescale.com/api/latest/compression/alter_table_compression/)
+for more detailed information about these queries.
```

Comment:
For the data compression paragraph, I would first explain better the steps, then give the commands. For example:

``````markdown
## Enable data compression

The first step is to turn on data compression on the target table, and set the compression options. Refer to the [TimescaleDB documentation](https://docs.timescale.com/api/latest/compression/alter_table_compression/) for a full list of options.

```sql
ALTER TABLE <target-table> SET (timescaledb.compress, timescaledb.compress_segmentby = '<id-column>', timescaledb.compress_orderby = '<name-column>'
```

Then you have to create the compression policy. The interval determines the age that the chunks of data need to reach before being compressed. Read the [official documentation](https://docs.timescale.com/api/latest/compression/add_compression_policy/) for more information.

```sql
SELECT add_compression_policy('<target-table>', INTERVAL '<time>');
```

``````

Reply:
updated